src/MusicStore/Controllers/ManageController.cs

```diff
@@ -13,19 +14,15 @@ namespace MusicStore.Controllers
     [Authorize]
     public class ManageController : Controller
     {
-        public ManageController(UserManager<ApplicationUser> userManager, SignInManager<ApplicationUser> signInManager)
-        {
-            UserManager = userManager;
-            SignInManager = signInManager;
-        }
+        [Activate]
+        public UserManager<ApplicationUser> UserManager { get; set; }
 
-        public UserManager<ApplicationUser> UserManager { get; private set; }
-
-        public SignInManager<ApplicationUser> SignInManager { get; private set; }
+        [Activate]
+        public SignInManager<ApplicationUser> SignInManager { get; set; }
 
         //
         // GET: /Manage/Index
-        public async Task<ActionResult> Index(ManageMessageId? message = null)
+        public async Task<ActionResult> Index(CancellationToken requestAborted, ManageMessageId? message = null)
```

Comment:
Maybe just call this `CancellationToken cancellationToken`? It's the common nomenclature for CancellationToken.